src/main/java/com/vadmack/mongodbtest/log/RequestLogger.java

```diff
@@ -0,0 +1,61 @@
+package com.vadmack.mongodbtest.log;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StopWatch;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Component
+@Aspect
+public class RequestLogger {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    @Pointcut("@annotation(LogRequests)")
+    public void annotation() {
+    }
+
+    @Around("annotation()")
```

Comment:
You can define an pointcut as annotation directly in the `@Around`. So, `annotation()` method is not needed